management/src/main/scala/io/scalac/mesmer/management/service/ActorTreeRoutesProvider.scala

```diff
@@ -0,0 +1,153 @@
+package io.scalac.mesmer.management.service
+
+import akka.actor.ExtendedActorSystem
+import akka.actor.setup.Setup
+import akka.actor.typed.scaladsl.AskPattern._
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.scaladsl.adapter._
+import akka.actor.typed.{ ActorRef, ActorSystem }
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import akka.management.scaladsl.{ ManagementRouteProvider, ManagementRouteProviderSettings }
+import akka.util.Timeout
+import akka.{ actor => classic }
+import com.typesafe.config.Config
+import io.scalac.mesmer.core.model.Tag
+import io.scalac.mesmer.extension.config.Configuration
+import io.scalac.mesmer.extension.service.ActorTreeService.Command.GetActors
+import io.scalac.mesmer.extension.service.{ actorTreeServiceKey, ActorTreeService, NonEmptyTree }
+import io.scalac.mesmer.extension.util.GenericBehaviors
+import io.scalac.mesmer.management.service.ActorInfoService.{ actorInfo, ActorInfo }
+import org.slf4j.LoggerFactory
+import zio.Chunk
+import zio.json._
```

Comment:
It looks like Akka Management depends on akka-http-spray-json, so we might not need any other dependency at all https://repo1.maven.org/maven2/com/lightbend/akka/management/akka-management_2.13/1.1.0/akka-management_2.13-1.1.0.pom